WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameView.swift

```diff
@@ -62,6 +62,8 @@ class SiteNameView: UIView {
         WPStyleGuide.configureSearchBar(searchBar, backgroundColor: .clear, returnKeyType: .continue)
         searchBar.setImage(UIImage(), for: .search, state: .normal)
         searchBar.autocapitalizationType = .sentences
+        searchBar.accessibilityIdentifier = "Website Title"
+        searchBar.searchTextField.attributedPlaceholder = NSAttributedString()
```

Comment:
This had me perplexed because I didn't understand why `placeholder` would depend on `layoutSubviews` (I'm assuming that's what you meant above).

It turns out it's because we're setting the [style globally here](https://github.com/wordpress-mobile/WordPress-iOS/blob/91f3525592fd25b5fc580571738506201aae930a/WordPress/Classes/Extensions/Colors%20and%20Styles/WPStyleGuide%2BSearch.swift#L52). 😬 

In the future I wonder if simply making a subclass would avoid this?